source/win_evaluation_poker.py

```diff
@@ -0,0 +1,83 @@
+import CardGames
+from Find_Flush import *
+from FourOfAKind import *
+from Find_FullHouse import *
+from PokerPlayer4Functions import *
+from Straight_Hand import *
+
+class win_evaluation_poker:
+
+    def __init__(self, playerlist, river, betting):
+        player=Player("test", 10)
+        player.hand_value=(10, 0)
+        self.winner=player
+        self.winnerList=[]
+        self.players=playerlist
+        self.river=river
+        self.betting=betting
+
+    def find_hand_values(self, playerlist, river):
```

Comment:
I'm glad all of our hand functions fit together here. Hopefully you didn't have to adjust too much.

Reply:
Not really, just had to make sure all the functions outputted the same things